runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java

```diff
@@ -186,10 +187,15 @@ public void onChatMessage(ChatMessage chatMessage)
 			{
 				messageNode.setValue(matcher.replaceAll(usernameReplacer));
 				update = true;
-
-				if (config.notifyOnOwnName())
 				{
```

Comment:
You have a random block starting here for no reason now. Get rid of this and the brace on L199.
```suggestion

```